seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-split/src/main/java/org/apache/seatunnel/flink/transform/Split.java

```diff
@@ -20,21 +20,19 @@
 import org.apache.seatunnel.common.config.CheckConfigUtil;
 import org.apache.seatunnel.common.config.CheckResult;
 import org.apache.seatunnel.flink.FlinkEnvironment;
-import org.apache.seatunnel.flink.batch.FlinkBatchTransform;
 import org.apache.seatunnel.flink.stream.FlinkStreamTransform;
 
 import org.apache.seatunnel.shade.com.typesafe.config.Config;
 
 import org.apache.flink.api.common.typeinfo.TypeInformation;
-import org.apache.flink.api.java.DataSet;
 import org.apache.flink.api.java.typeutils.RowTypeInfo;
 import org.apache.flink.api.scala.typeutils.Types;
 import org.apache.flink.streaming.api.datastream.DataStream;
 import org.apache.flink.types.Row;
 
 import java.util.List;
 
-public class Split implements FlinkStreamTransform, FlinkBatchTransform {
+public class Split implements FlinkStreamTransform {
 
     private Config config;
```

Comment:
About the Transform refactor:

- Could you remove the FlinkStreamTransform? 
- rename the method name from processStream to process
- move the process to BaseFlinkTransform
- Change the BaseFlinkTransform from interface to abstract class
- move the `private Config config` and setter/getter to BaseFlinkTransform

After finish these changes, it can greatly reduce the duplication of code and remove the useless class.